atc/metric/emitter/influxdb.go

```diff
@@ -24,9 +26,22 @@ type InfluxDBConfig struct {
 	Password string `long:"influxdb-password" description:"InfluxDB server password."`
 
 	InsecureSkipVerify bool `long:"influxdb-insecure-skip-verify" description:"Skip SSL verification when emitting to InfluxDB."`
+
+	// https://github.com/influxdata/docs.influxdata.com/issues/454
```

Comment:
Thanks for giving us the context on "why 5000"! That's very helpful.

Wdyt of leaving that to the PR though? As one could always trace the `5000` back to this PR, I'd say leaving that in the PR's comments (or even in the commit message) would be enough 🤔 In the codebase, we don't have many other cases where we add references like this.

Reply:
I will remove the comments and add as a comment here:

https://github.com/influxdata/docs.influxdata.com/issues/454
https://docs.influxdata.com/influxdb/v0.13/write_protocols/write_syntax/#write-a-batch-of-points-with-curl
5000 seems to be the batch size recommended by the InfluxDB team